android/src/main/java/com/reactnativecommunity/cameraroll/CameraRollModule.java

```diff
@@ -313,6 +314,12 @@ protected void doInBackgroundGuarded(Void... params) {
       }
       WritableMap response = new WritableNativeMap();
       ContentResolver resolver = mContext.getContentResolver();
+
+      // TODO: Remove this when updating the project.
+      // DATE_TAKEN can result in images having a timestamp of zero on Android 10+, hence we use DATE_ADDED
+      // in that case.
+      String mediaQuery = Build.VERSION.SDK_INT >= 29 ? Images.Media.DATE_ADDED : Images.Media.DATE_TAKEN;
```

Comment:
Can't we make use of the `DATE_ADDED` for those images that do have it available?

something like: `Images.Media.DATE_TAKEN + " DESC, " + Images.Media.DATE_ADDED + " DESC, " + ...`